extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSFilter.java

```diff
@@ -90,7 +90,8 @@ public void handle(RoutingContext event) {
                 processRequestedHeaders(response, requestedHeaders);
             }
 
-            boolean allowsOrigin = corsConfig.origins.isEmpty() || corsConfig.origins.contains(origin);
+            boolean isWildcardOrigin = corsConfig.origins.size() == 1 && "*".equals(corsConfig.origins.get(0));
```

Comment:
Hmmm, isn't it a breaking change? If it's empty, it used to pass, now it doesn't. Is it really what we want? Or did I miss something.

Blocking merge until I got an answer to that.

Reply:
Good catch, `corsConfig.origins.isEmpty()` should still be there.